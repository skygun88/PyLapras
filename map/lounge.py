import os
import pandas as pd

PYLAPRAS_PATH = os.path.abspath(os.path.dirname(__file__)).split('PyLapras')[0]+'PyLapras'

def load_map():
    df = pd.loa