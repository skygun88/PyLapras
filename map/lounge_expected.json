{
    "Activity": {
        "S01": [53.07, 46.25],
        "S02": [51.65, 45.14],
        "S03": [51.10, 48.57],
        "S04": [52.05, 48.60],
        "S05": [52.90, 48.68],
        "S06": [51.12, 47.71],
        "S07": [52.15, 47.79],
        "S08": [52.88, 47.76],
        "S09": [51.17, 50.28],
        "S10": [52.42, 50.41]
    },
    "Motion": {
        "M01": [52.00, 44.96],
        "M02": [52.00, 49.09],
        "M03": [52.76, 51.77],
        "M04": [50.17, 44.89],
        "M07": [50.04, 49.03],
        "M08": [50.01, 51.46]
    }
}